frontend/model/contracts/mailbox.js

```diff
@@ -41,9 +50,9 @@ sbp('chelonia/defineContract', {
         const initialState = merge({
           attributes: {
             creator: data.username,
-            autoJoinAllowance: true
+            autoJoinAllowance: true // this attribute could be used to block him to be joined direct/group messages automatically by another
```

Comment:
Could you please clarify what this attribute is for? The comment still isn't very clear to me